diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter1/PackageNamingWarnTest.kt

```diff
@@ -71,6 +71,24 @@ class PackageNamingWarnTest : LintTestBase(::PackageNaming) {
         )
     }
 
+    @Test
+    @Tag(WarningNames.PACKAGE_NAME_MISSING)
+    fun `don't add the package name to the file in buildSrc path`() {
+        lintMethod(
+            """
+                import org.cqfn.diktat.a.b.c
+
+                /**
+                 * testComment
+                 */
+                class TestPackageName {  }
+
+            """.trimIndent(),
+            fileName = "~/diktat/diktat-rules/src/nativeMain/kotlin/org/cqfn/diktat/buildSrc/BlaBla.kt",
```

Comment:
Please use a more standard path to `buildSrc`

Reply:
Fixed